x-pack/plugins/integration_assistant/server/plugin.ts

```diff
@@ -77,9 +79,7 @@ export class IntegrationAssistantPlugin
     const { licensing } = dependencies;
 
     licensing.license$.subscribe((license) => {
-      if (!license.hasAtLeast('enterprise')) {
-        this.isAvailable = false;
-      }
+      this.hasLicense = license.hasAtLeast('enterprise');
```

Comment:
Can we use the `LicenseType` type here instead of hardcoding https://github.com/elastic/kibana/blob/369fb6028c6ea2f4f03fa2363949bfa42f2a2b4c/x-pack/plugins/licensing/common/types.ts#L10

Reply:
That'd be great, but the `hasAtLeast` function parameter is already typed with the union type of the license strings, I can not use the enum here:
```
Argument of type 'LICENSE_TYPE.enterprise' is not assignable to parameter of type '"basic" | "standard" | "gold" | "platinum" | "enterprise" | "trial"'.
```